client/src/components/LabeledTable.js

```diff
@@ -31,7 +31,7 @@ export class LabeledTable extends React.Component {
                   { styles && styles.desc ? <span className={styles.desc}>item.desc</span> : item.desc }
```

Comment:
Can you just wrap `item.desc` to `{item.desc}`? this seems like 'potential' bug. Same with line 39